templates/source/sourceTemplate.md

```diff
@@ -0,0 +1,35 @@
+# Event Source for **\_\_\_\_**
+
+This event target receives .......
```

Comment:
The title says "source", templates should be consistent.

Reply:
Yes, sorry. Targets were stuck on my brain. I will remove the "target" here.